pdfmaker/kassia.py

```diff
@@ -43,17 +43,19 @@ def parseFile(self):
             
     def createPDF(self):
         """Create PDF output file"""
-        # TODO: Parse page layout and formatting
+        # Parse page layout and formatting
+        if (self.bnml is not None):
+            margin_attrib = self.bnml.attrib
+            margin_attrib = self.fill_page_dict(margin_attrib)
+
         self.paperSize = letter
-        self.topmargin = 72
-        self.bottommargin = 72
-        self.leftmargin = 72
-        self.rightmargin = 72
-        self.lineHeight = 72
-        self.lineWidth = self.paperSize[0] - (self.leftmargin + self.rightmargin)
-        self.nFontSize = 20
-        self.lFontSize = 12
-        
+        self.topMargin = margin_attrib['top_margin']
+        self.bottomMargin = margin_attrib['bottom_margin']
+        self.leftMargin = margin_attrib['left_margin']
+        self.rightMargin = margin_attrib['right_margin']
+        self.lineHeight = margin_attrib['line_height']
+        self.lineWidth = self.paperSize[0] - (self.leftMargin + self.rightMargin)
+
```

Comment:
I like the abstraction, but these need reasonable defaults if they're left unspecified.